### ISSUE TITLE
Use this script with non-Cyanogenmod XT926?

### ISSUE THREAD

**Issue by SeaDude** [stargo/xt897-fix-usa#1]
Hello. 
Is there a way to use your script as outlined in t[his XDA post](https://forum.xda-developers.com/showpost.php?p=54773773&postcount=7), on an XT926 **without** Cyanogenmod (Android 4.4.2)?

**Comment by stargo** [comment]
Yes, it should work on the stock rom (and all other roms) out of the box.
